inc/Services/Theme.php

```diff
@@ -74,4 +82,41 @@ private function i18n(): void {
 		// Load theme texdomain
 		load_theme_textdomain( 'framework-textdomain', \get_theme_file_path( '/languages' ) );
 	}
+
+	/**
+	 * Set default path for ARI for minified files
+	 *
+	 * @param string $attr
+	 *
+	 * @return string
+	 *
+	 */
+	public function set_ari_responsive_image_default_img_path( string $attr ): string {
+		return '/dist/';
+	}
+
+	/**
+	 * Set ari default image name for minified files
+	 *
+	 * @param string $default_img
+	 *
+	 * @return string
+	 *
+	 */
+	public function set_ari_responsive_image_default_img_name( string $default_img ): string {
+		return $this->get_min_default_image( $default_img );
+	}
+
+	/**
+	 * Get minified default_image
+	 *
+	 * @param string $original_image
+	 *
+	 * @return string
+	 *
+	 * @author Léonard Phoumpakka
+	 */
+	public function get_min_default_image( string $original_image ): string {
+		return Framework::get_container()->get_service( 'assets' )->get_min_file( 'assets/' . $original_image );
+	}
```

Comment:
Ici on peut utiliser l'instance du service `Assets` qui est stocké dans `$this->assets`.